tutorials/23_MultiHop_QA_With_Agents.ipynb

```diff
@@ -0,0 +1,5680 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "h8gcxwg4sAXd"
+      },
+      "source": [
+        "# Tutorial: Multihop Question Answering with Agents\n",
+        "\n",
+        "- **Level**: Advanced\n",
```

Comment:
I'd say it's intermediate. I could easily follow it. And then when we have a tutorial with multiple tools, this could be advanced, but up to you.